storage/__init__.py

```diff
@@ -3,5 +3,5 @@
 # you may not use this file except in compliance with the Elastic License 2.0.
 
 from .factory import StorageFactory
-from .s3 import CHUNK_SIZE, S3Storage
-from .storage import CommonStorage
+from .s3 import S3Storage
+from .storage import CHUNK_SIZE, CommonStorage, StorageReader
```

Comment:
This change makes the `CHUNK_SIZE` smaller (from `1024 ** 2` in `.s3` to `1024` in `.storage`).

Is this change intended? I'm curious why `1024` is better than `1024 ** 2` in this context.

Reply:
`CHUNK_SIZE` is how much we read from the stream at every iteration. lowered from 1M to 1K since it will have less impact on memory and produce smaller iteration.
it doesn't seem to have any impact on speed (current `TestS3Storage.test_get_by_lines` takes around 10 sec with both values)

we will tweak the best value once we will have info about production data (avg/min/max filesize etc)